AWSAppSync.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.dependency 'AWSCore', '~> 2.12.0'
   s.dependency 'SQLite.swift', '~> 0.12.2'
   s.dependency 'ReachabilitySwift', '~> 5.0.0'
-  s.dependency 'Starscream', '~> 3.0.2'
```

Comment:
Will this work without the Starscream?

Reply:
yup, since the dependency AppSyncRealTimeClient depends on Starscream. taking on the AppSyncRealTimeClient dependency is enough